src/main/java/service/ReportWriter.java

```diff
@@ -0,0 +1,5 @@
+package service;
+
+public interface ReportWriter {
```

Comment:
let it be more abstract
```suggestion
public interface FileWriter {
```